00 Meta/Contribution.md

```diff
@@ -0,0 +1,63 @@
+# How to contribute
+
+First of all, thanks for considering sharing your knowledge! For adding a query to the vault, there are two things to do: First, **add the query** with fitting test data so it produces results and secondly, **share the query** so it'll become part of the example vault.
+
+## Get a local copy
+
+First of all, you need to get yourself a local copy of the example vault. There are two ways of doing this.
+
+1. Get a static local copy. Follow the [[README#Usage]] explanation for that
```

Comment:
Missing period at end of line ;)

Reply:
Also: perhaps it might not be clear to others what "static" means.

Reply:
Mhh, good point. I'll just throw it out for now.